apps/webpack.js

```diff
@@ -48,7 +48,7 @@ var baseConfig = {
           path.resolve(__dirname, 'test'),
           path.resolve(`${__dirname}/../dashboard/app/assets/`, 'images'),
         ],
-        loader: "url-loader?limit=1024&name=[name]-wp-[hash].[ext]",
+        loader: "url-loader?limit=1024&name=[name]wp[hash].[ext]",
```

Comment:
For posterity: it appears that rails doesn't like to serve up files that end with `-<hex digest>` if it is not currently configured to digest files itself. But if you get rid of the `-` then it works. Rails makes me 😢 .